eng/CredScanSuppression.json

```diff
@@ -44,6 +44,12 @@
                 "tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py"
             ],
             "_justification": "File contains private key used by test code."
+        },
+        {
+            "hash":[
+                "sHdLtMfSDs4w/A83iF5BEf3+19x3is6z4yqFHtkxKLs="
+            ],
+            "_justification": "Password in docstrings"
```

Comment:
For extra context can you add the string format for the docstrings that is being suppressed.